scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/locations/Locations.kt

```diff
@@ -31,7 +31,7 @@ class Locations @Inject constructor(
 
     val menuSelectQuestClick =
         (if (isWide)
-            Location(-410, 440)
+            Location(-600, 440)
```

Comment:
Hey, thanks for the PR.
I just have a concern about changing this value by too much.
We had some issues in past related to this which I'm not able to find right now.
There was some raid event which had a delay in opening AP refill dialog, we had to shift the click to be between the AP dialog and the node info button to solve that.
See if you can get by with a little less change in the value.

Reply:
Did some quick testing with other values, `-460` seems to be fairly close to the minimum, and is outside the AP box on my device at least.